lib/Dialect/FIRRTL/Transforms/VBToBV.cpp

```diff
@@ -732,7 +732,7 @@ Attribute Visitor::convertVectorConstant(FVectorType oldType,
     newElements.push_back(convertConstant(oldElementType, oldElement));
   }
 
-  auto bundleType = newElementType.cast<BundleType>();
+  auto bundleType = dyn_cast<BundleType>(newElementType);
```

Comment:
was this `cast` to `dyn_cast` intentional ?

Reply:
Thanks! It seems mistake 😔